src/prettier/rules/husky/apply-husky-configuration.ts

```diff
@@ -0,0 +1,10 @@
+import { Rule, Tree } from '@angular-devkit/schematics';
+import { createHuskyRc } from './create-husky-rc';
+import { updateHuskyRc } from './update-husky-rc';
+
+export function applyHuskyConfiguration(): Rule {
+  return (tree: Tree) =>
+    !tree.exists('.huskyrc.json')
```

Comment:
also check, if husky entry exists in package.json

Reply:
👍 I introduced a general rule to our lib. Now it can be reused for each package.

We may want to add it to commitlint as well. 😉

Reply:
;-)

Reply:
:wink: